Mathlib/Topology/Basic.lean

```diff
@@ -27,13 +26,11 @@ partially defined functions.
 
 ## Notation
 
+The following notation is introduced elsewhere and it heavily used in this file.
+
 * `𝓝 x`: the filter `nhds x` of neighborhoods of a point `x`;
 * `𝓟 s`: the principal filter of a set `s`;
 * `𝓝[s] x`: the filter `nhdsWithin x s` of neighborhoods of a point `x` within a set `s`;
-* `𝓝[≤] x`: the filter `nhdsWithin x (Set.Iic x)` of left-neighborhoods of `x`;
-* `𝓝[≥] x`: the filter `nhdsWithin x (Set.Ici x)` of right-neighborhoods of `x`;
-* `𝓝[<] x`: the filter `nhdsWithin x (Set.Iio x)` of punctured left-neighborhoods of `x`;
-* `𝓝[>] x`: the filter `nhdsWithin x (Set.Ioi x)` of punctured right-neighborhoods of `x`;
 * `𝓝[≠] x`: the filter `nhdsWithin x {x}ᶜ` of punctured neighborhoods of `x`.
```

Comment:
Hmmm, I get why you're doing that, but maybe we want to keep all of these together somewhere. I recall a long time ago there was some discussion about a documentation note listing all frequently-used filters so maybe it would belong there, but for now I think I prefer to keep the full list here.

Reply:
All these are listed in the new file `Topology/Defs/Filter`. I can revert this change as well, if you want.

Reply:
Sorry, ignore my comment then.